client/state/ui/editor/post/actions.js

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
I don't think we should put this under UI. I think several (most?) of these actions should be in the `posts` sub-tree. Anything else that is intrinsically specific to editor could be in a new `state.editor` portion.